src/test/java/com/epam/automation/inside.java

```diff
@@ -1,136 +0,0 @@
-        // (ez alól kivétel Amerika, ahol USA-t kell írni az ország ismétlése helyett)
-        driver.findElement(By.xpath("//*[starts-with(@id,'select-box-location-')]")).click();
-        driver.findElement(By.cssSelector("[aria-label=\"" + country + "\"]")).click();
-        if (city == country) {
-            driver.findElement(By.xpath("//*[contains(@id, 'all_" + city + "')]")).click();
-        } else {
-            driver.findElement(By.xpath("//*[contains(@id, '" + city + "')]")).click();
-        }
-        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
-    }
-
-
-    public void openSkillTab() {
-        /*külön vettem egyelőre a lista megnyitást és a skillek kiválasztását, mert több skill kiválasztása esetén errort ad, mivel 2xer akarja megnyitni.
-        if-el vizsgáltam htmlként, hogy van e függőleges lista megnyitva, de a keywords input rész megzavarja az ajánlott keywords kiegészítésekkel, így
-        false infót ad az állapotról
-        */
-        driver.findElement(By.xpath("/html/body/div[1]/div[3]/div[1]/div[1]/section/div/div[4]/form/div/div[2]/div/div[1]/div[1]")).click();
-        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
-    }
-
-    public void skills(String skill) {
-        driver.findElement(By.xpath("//*[@class='checkbox-custom-label' and contains(., '" + skill + "')]")).click();
-        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
-    }
-
-    public void searchButton() {
-        driver.findElement(By.xpath("/html/body/div[1]/div[3]/div[1]/div[1]/section/div/div[4]/form/div/div[3]/button")).click();
-        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
-    }
-
-    public void showJobsWithLocation() {
-        //kilistázza a munkákat munkavégzés helyével együtt
-        String jobs = "";
-        String loc = "";
-        List<WebElement> job = driver.findElements(By.className("search-result__item-name"));
-        for (WebElement ele : job) {
-            jobs = ele.getText();
-
-            List<WebElement> location = driver.findElements(By.className("search-result__location"));
-            for (WebElement ele2 : location) {
-                loc = ele2.getText();
-            }
-            System.out.println(jobs + " " + loc);
-        }
-        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
-    }
-
-    public void sortJobsByDate() throws  Exception {
-    try {
-        driver.findElement(By.cssSelector("li.search-result__sorting-item:nth-child(2)")).click();
-        /*egyelőre Thread.sleepet használok, mert az implicitlyWaitnél vagy a sleep nélküli állapotnál folyamatosan
-        StaleElementReferenceException errort kapok. Az oldal az update óta lassan frissíti a munkákat dátum alapján, ezért
-        a listázás közben történt a frissítés, ami errort okoz a tesztben. A try/catch blokkot benne hagytam biztonság kedvéért.
-         */
-        Thread.sleep(1000);
```

Comment:
Never use sleep in your tests, explicit waits are much more efficient and elegant.